proposals/adrianhopebailie-201908.md

```diff
@@ -0,0 +1,588 @@
+---
+Change Request ID: ???
```

Comment:
Can we use the ID of the issue https://github.com/mojaloop/mojaloop-specification/issues/18 ?